Charts/Change-X-axis-label-position/.NET-Standard/Change-X-axis-label-position/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.IO;
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.OfficeChart;
+
+namespace Change_X_axis_label_position
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (FileStream fileStream = new FileStream(Path.GetFullPath(@"../../../Data/Input.docx"), FileMode.Open))
+            {
+                //Loads an existing Word document.

```

Comment:
Remove unwanted empty lines.
Maintain uniformity in XML comments. (Some starts with 's and some not)
XML comment should end with full stop.

Reply:
Feedbacks are addressed